notebooks/dataframes/integrations.ipynb

```diff
@@ -66,9 +66,21 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        token = bqclient._http.credentials.token
</pre><p>FYI: This is the only <em>code</em> line that changes. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/1784/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>